addons/arrays/test_shuffle.sqf

```diff
@@ -21,18 +21,14 @@ TEST_OP(count _result,==,count _original,_fn);
     TEST_OP(_x,in,_original,_fn);
 } forEach _result;
 
-// Test depecated version.
+// test modify original
```

Comment:
`[1,2,3,4,5,6,7] call CBA_fnc_shuffle; `
Old version returns: [4,7,2,1,3,5,6]
New version returns: []

Old version threw a warning to rpt, but still worked.
I am a little worried some people would use it like that.